project/src/molecules/constructor.py

```diff
@@ -0,0 +1,33 @@
+"""Methods used to create systems from job statepoint."""
+import mbuild as mb
+import signac
+from mbuild import Box
+
+from project.src import molecules
+
+from .methane_ua import MethaneUA
+
+
+def SystemBuilder(job):
+    """Construct systems according to job statepoint."""
+    # Update this dict as new reicpes are made
+    molecule_dict = {
+        "methaneUA": MethaneUA,
+        "pentaneUA": None,
+        "benzeneUA": None,
+        "waterSPC/E": None,
+        "ethanolAA": None,
+    }
+    molecule = molecule_dict[job.sp["molecule"]]
+    vap_box = mb.Box()
+    liq_box = mb.Box()
+
+    vap_box_cpd = mb.fill_box(
+        compound=molecule, n_compound=job.sp["n_compounds"], box=vap_box
+    )
+    liq_box_cpd = mb.fill_box(
+        compound=molecule, n_compound=job.sp["n_compounds"], box=liq_box
+    )
```

Comment:
Would "n_compounds" be split between the two boxes?

Reply:
There are now separate N_liq and N_vap variables in the job statepoints. If vapor phase is not needed, it is Null in the project statepoint. I think signac accesses that as `None`

Reply:
This info is split into 2 in the new job statepoint set up.